docs/standard/library.md

```diff
@@ -1,105 +1,124 @@
 ---
-title: .NET Standard Library
-description: .NET Standard Library
-keywords: .NET, .NET Core
+title: .NET Standard | Microsoft Docs
+description: Learn what is .NET Standard, its versions and the .NET platforms supported.
+keywords: .NET Standard, PCL, .NET
 author: richlander
 ms.author: mairaw
-ms.date: 06/20/2016
+ms.date: 03/16/2017
 ms.topic: article
 ms.prod: .net
 ms.technology: dotnet-standard
 ms.devlang: dotnet
 ms.assetid: c044882c-af15-45f2-96d1-534557a5ee9b
 ---
 
-# .NET Standard Library
```

Comment:
I will add a comment on this since this was a big source of confusion for me.  
A "library" is code you execute.
A "standard" is something that describes what you can expect the library to do.
The term "standard library" is a misnomer in this context since it mixes those two concepts.
For this converstation the libraries are .Net core, .Net Framework, etc.  There is no "standard library".
I can understand the term "Standard Libraries" which may mean all the libraries that implment the standard but I think care should be taken not to imply there is a library different from the others in that it is itself the standard.